parsing/dialects/dialect1/config.fmpp

```diff
@@ -1024,6 +1026,7 @@ data: {
     # Return type of method implementation should be "SqlTypeNameSpec".
     # Example: SqlParseTimeStampZ().
     dataTypeParserMethods: [
+      "SqlPeriodDataType()"
```

Comment:
nit: While you're here, can you reorder the dataTypeParserMethods to be in alphabetical order?

Reply:
Changed to alphabetical order in https://github.com/googleinterns/calcite/pull/177/commits/110d83e563426a9ea05113263c6efc02c44447dc

Reply:
Thanks!